Source/Synthesis.Tests/Fixtures/ContentSearch/ContentSearchQueryExtensionTests.cs

```diff
@@ -45,5 +48,12 @@ public void ContentSearch_FindsSecurityFolderOrRenderingOptions_WhenContainsOrIs
 				Assert.IsTrue(query.Any(x => x.Name == listOfNames[1]), "ContainsOr result contained no rendering options item");
 			}
 		}
+
```

Comment:
This should be unnecessary once we add a DI model to `GetSynthesisQueryable` (see https://github.com/kamsar/Synthesis/pull/21/files#r32533998)